src/common/config.cc

```diff
@@ -248,6 +248,9 @@ int md_config_t::parse_config_files(const char *conf_files,
       } else {
 	cfl.erase(p++);  // ignore this item
       }
+    } else if (s.find("$home") != string::npos) {
+	s.replace(0, 5, string(getenv("HOME")));
```

Comment:
Would it be more appropriate to move this part to somewhere in expand_meta()?

Reply:
@rishabh-d-dave 
I think that would make sense, it would then be more universially applicable.